docs/constants-and-vars.rst

```diff
@@ -11,11 +11,12 @@ Environment variables always exist in the namespace and are primarily used to pr
 Block and Transaction Properties
 --------------------------------
 
-==================== ================ =============================================
+==================== ================ =========================================================
 Name                 Type             Value
-==================== ================ =============================================
+==================== ================ =========================================================
 ``block.coinbase``   ``address``      Current block miner’s address
 ``block.difficulty`` ``uint256``      Current block difficulty
+``block.prevrandao`` ``uint256``      Current block difficulty (alias for ``block.difficulty``)
```

Comment:
```suggestion
``block.prevrandao`` ``uint256``      Current slot randomnes beacon
```

Reply:
I made it even more clear here: https://github.com/vyperlang/vyper/pull/3085/commits/1e30b316554048eb6015605ff9ff8207f16012a5